changes/340.documentation

```diff
@@ -0,0 +1 @@
+Added example to documentation showing how to assign an IP address to an interface.
```

Comment:
Copy pasta?

Reply:
What should have been better, but so much worse.